modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js

```diff
@@ -78,12 +57,14 @@ const ListItems = ({defaultLanguageId, itemType, items}) => {
 						<Cell align="left">
 							{getLocalizedValue(defaultLanguageId, name)}
 						</Cell>
-						<Cell>{fromNow(dateCreated)}</Cell>
-						<Cell>{fromNow(dateModified)}</Cell>
+						<Cell>{dateCreated && fromNow(dateCreated)}</Cell>
+						<Cell>{dateModified && fromNow(dateModified)}</Cell>
 						<Cell align={'right'}>
 							<ClayRadioGroup
 								inline
-								onSelectedValueChange={() => onItemIdChange(id)}
+								onSelectedValueChange={(id) =>
+									onItemIdChange(id)
+								}
```

Comment:
Hey @gabrielpramos , what do you think about just call the `onChange` function? 

Like this:
`onSelectValueChange={onChange}`